legacy-sdk/whirlpool/tests/sdk/whirlpools/client-impl/whirlpool-impl.test.ts

```diff
@@ -1628,8 +1629,12 @@ describe("whirlpool-impl", () => {
 
     // check generated instructions
     const instructions = tx.compressIx(true).instructions;
-    const createIxs = instructions.filter((ix) =>
-      ix.programId.equals(ASSOCIATED_PROGRAM_ID),
+    const createIxs = instructions.filter(
+      (ix: {
```

Comment:
smol nit: is there a reason you had to provide the type now? and if so, is there not an (anchor) type for this so that we don't have to define ourselves?